package.json

```diff
@@ -97,6 +97,12 @@
           "default": false,
           "description": "Show Tekton Pipeline output channel when new text added to output stream"
         },
+        "vs-tekton.showPipelineRunFollowLogs": {
+          "title": "Show pipelineRun follow Logs",
```

Comment:
I think it would be better to have preference for regular start pipeline command, something like `Show logs on pipeline start/run`, instead of having separate command. Or at least title of this command can be changed to: `Start Pipeline with logs`.
@mohitsuman WDYT?

Reply:
@mohitsuman ???

Reply:
I think having vscode setting preference on the pipeline run and logs makes sense. And for the title `Start Pipeline with logs` works.